### PR TITLE
Update CAPI v1.6 Release Team usergroup members list

### PR DESCRIPTION
**Pull request by furkatgofurov7**
This PR updates the slack user groups for the new Cluster API release v1.6 team.

### CODE REVIEW

**Review by vincepri — APPROVED** [review]
/approve
/lgtm

File: communication/slack-config/users.yaml [hunk 1]
```diff
@@ -11,7 +11,6 @@ users:
   alisondy: U9CBCBLCV
   ameukam: U68KPQ448
   anandrkskd: U01DRS3V0R2
-  aniruddha2000: U02BU1RP41H
```
Comment (fabriziopandini) [inline communication/slack-config/users.yaml:14]: I don't think we should edit this file at all, it doesn't seem related to the release team membership

> This file contains an alphabetically sorted mapping of lowercase GitHub usernames to Kubernetes Slack user IDs.

Might be @joekr, @sbueringer or @killianmuldoon have more context here

Comment (sbueringer) [reply communication/slack-config/users.yaml:14]: No context. I assumed a presubmit failed and we regenerated it (?)

Comment (sbueringer) [reply communication/slack-config/users.yaml:14]: We definitely edited the file the last time we did this: https://github.com/kubernetes/community/pull/7230/files#diff-2812ad300173ab2701987b4ab3aecb9d79bb5bba665c683b26e9602a4cdbb062


> Users
> There is no safe, stable way to derive a specific Slack user from any human-readable identifier. Instead of using Slack IDs everywhere, a single mapping from GitHub usernames to Slack IDs is given in users.yaml. To reference a user, they must first be added to users.yaml.

Comment (sbueringer) [reply communication/slack-config/users.yaml:14]: Maybe we can keep the old mappings even though they are now not required anymore?

Comment (furkatgofurov7) [reply communication/slack-config/users.yaml:14]: checked the history of the file and looks like it has not been used to remove the mappings much, but only for adding, so reverting back deletions in the file

Comment (furkatgofurov7) [reply communication/slack-config/users.yaml]: @fabriziopandini @sbueringer updated

File: communication/slack-config/usergroups.yaml [hunk 2]
```diff
@@ -246,21 +246,22 @@ usergroups:
     description: Members of the Cluster API Release Team
     members:
       - adilGhaffarDev
-      - aniruddha2000
+      - cahillsf
       - CecileRobertMichon
```
Comment (CecileRobertMichon) [inline communication/slack-config/usergroups.yaml:250]: same question here as the github groups, should we remove @sbueringer and I since @vincepri is the maintainer for this release

Comment (sbueringer) [reply communication/slack-config/usergroups.yaml:250]: Fine for me. Same question for yourself? :)

Comment (sbueringer) [reply communication/slack-config/usergroups.yaml:250]: Sounds good to me

Comment (furkatgofurov7) [reply communication/slack-config/usergroups.yaml]: > same question here as the github groups, should we remove @sbueringer and I since @vincepri is the maintainer for this release

Done

**Review by vincepri — APPROVED** [review]
/approve

**Comment by Priyankasaggu11929** [comment]
/lgtm
/approve

Thanks!
